### ISSUE TITLE
Add "image description" field

### ISSUE THREAD

**Issue by alanmoo** [mozilla/network-pulse-api#256]
For accessibility purposes. Once this is done, we can use that description as the alt tag of images on the client side.

**Comment by Pomax** [comment]
Closing this issue due to having gone stale - if this is still something we want to do, let's file a new issue with all the information necessary for triaging.
